release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -258,8 +258,8 @@ To evict pods instead of simulating the evictions, change the descheduler mode t
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
 [id="ocp-4-11-logging"]
-=== Red Hat OpenShift Logging
-
+=== Red Hat OpenShift Logging (Technical Preview)
+{product-title} 4.11 introduces  a new connector for the {rh-virtualization} API, adding automated log messages for all interactions  for installations and oVirt components in a cluster.
```

Comment:
* "API, adding" -> "API that adds"
* "for all interactions for installations and oVirt components" sounds a little funny to me with the two instances of "for". Not sure if the interactions part is necessary? Could it be "log messages for all RHV installations and oVirt components in the cluster" Not sure if adding RHV is right there or not, just tossing it out there.